lib/chef/knife/winrm.rb

```diff
@@ -29,6 +29,13 @@ class Winrm < Knife
 
       include Chef::Knife::WinrmCommandSharedFunctions     
 
+      FAILED_BASIC_HINT ||= "Hint: Please check winrm configuration 'winrm get winrm/config/service' AllowUnencrypted flag on remote server."
+      FAILED_NOT_BASIC_HINT ||= <<-eos.gsub /^\s+/, ""
```

Comment:
I'm fine with the `gsub` as well. There are occurrences of both `gsub`bing and left-justifying all over our code bases.